<?xml version="1.0"?>
<launch>
   <arg name="calibration_path_url" default="file://$(find opencv_dnn)/config/calibration.yaml"/>
   <arg name="camera_name" default="camera"/>
   <arg name="from_tf" default="rgb_cam"/>
   <arg name="to_tf" default="ship_ref_point"/>
   <arg name="cartPlot_confidence" default="50"/>
    <param name="/use_sim_time" value="true"/>


    <node pkg="buoy_projector" type="cam_info" name="rgb_camera_info" output="screen">
        <param name="calibration_path" value="$(arg calibration_path_url)"/>
        <param name="camera_name" value="$(arg camera_name)"/>
        <param name="/use_sim_time" value="true"/>

    </node>

    <node pkg="buoy_projector" type="projector.py" name="buoy_projection" output="screen">
        <param name="from_tf" value="$(arg from_tf)"/>
        <param name="to_tf" value="$(arg to_tf)"/>
        <param name="cartPlot_confidence" value="$(arg cartPlot_confidence)"/>
        <remap from="rgb_projections" to="cortix/sense/ros_interface/detection/cartesian_plots"/>
    </node>
</launch>